docs/guide/persistence_cache.md

```diff
@@ -70,7 +59,7 @@ ezpublish:
         # "site_group" refers to the group configured in site access
         site_group:
             # cache_pool is set to '%env(CACHE_POOL)%'
-            # env(CACHE_POOL) is set to 'cache.app'
+            # env(CACHE_POOL) is set to 'cache.app', a symfony service
```

Comment:
Great that you mentioned it, it's not obvious for everyone 👍  

WDYT about `a symfony service` -> `a Symfony service, for examples see app/config/cache_pool/*`